ui/tests/e2e/test.js

```diff
@@ -168,6 +168,27 @@ describe("End-to-end", () => {
     await waitForFacetsUpdate(3500);
   });
 
+  test("Search box - values", async () => {
+    // Type "pat" in search box.
+    let initial_select = await page.$x("//div[text()='Search center or pat']");
+    await initial_select[0].click();
+    await initial_select[0].type("pat");
+
+    // Select first result
+    await page.waitForXPath("//span[contains(text(), 'Add')]");
+    let results = await page.$x("//span[contains(text(), 'Add')]");
+    await results[0].click();
+
+    // Assert Relationship facet is added
+    await waitForFacetCard("Relationship");
+    await assertFacet("Relationship", "1", "mother", "831");
+
+    // Make sure non-selected facet values are gray.
```

Comment:
opt - These tests should really be asserting that a facet value is checked, not that a different value is gray. I could have sworn that was somewhere in this test, but I can't find it now. Up to you -- add assertFacetValueSelect() and have this test and others use it.

Reply:
I am not sure which other tests need this, I changed this test to check if the value is selected.